_includes/k8s-spec-generate.md

```diff
@@ -4,11 +4,40 @@ To generate the spec file for the 1.3 release, head on to [1.3 install page](htt
 
 Alternately, you can use curl to generate the spec as described in [Generating Portworx Kubernetes spec using curl](/scheduler/kubernetes/px-k8s-spec-curl.html).
 
->**Secure ETCD:**<br/> If using secure etcd provide "https" in the URL and make sure all the certificates are in the _/etc/pwx/_ directory on each host which is bind mounted inside PX container.
+#### Secure ETCD and Certificates
+If using secure etcd provide "https" in the URL and make sure all the certificates are in the _/etc/pwx/_ directory on each host which is bind mounted inside PX container.
 
+#### Using Secrets to Provision Certificates
+>**Note for ASG users:**<br/>When using PX with Auto Scaling Groups in AWS, you should use Secrets to have Kubernetes provision the ETCD certificates.  This way, the certificates will be provisioned on any new EC2 instance automatically.
 
-##### Installing behind the HTTP proxy
+Use `kubectl` to create the secret, for example:
+
+```
+# kubectl -n kube-system create secret generic etcd-certs --from-file=etcd-secrets/
+```
+
+Now edit the Portworx spec file to reference the certificates.  For example, assuming the names of the files are `pwx-ca.crt` `pwx-user-cert.crt` and `pwx-user-key.key`, modify the spec file as follows:
+
+```
+  - mountPath: /etc/pwx/etcdcerts
+    name: etcdcerts
+```
+
+```
+  - name: etcdcerts
+    secret:
+      secretName: etcd-certs
+      items:
+      - key: pwx-ca.crt
+        path: pwx-ca.crt
+      - key: pwx-user-cert.crt
+        path: pwx-user-cert.crt
+      - key: pwx-user-key.key
+        path: pwx-user-key.key
+```
+
```

Comment:
we should mention that the options that we will be using in the container params `-key = /etc/pwx/etccerts/pwx-ca.crt` and so on. At least a reminder under what path the secrets will be found in the pod (/etc/pwx/etccerts)

Reply:
Can you update the PR with these edits?